pydis_site/templates/resources/resources.html

```diff
@@ -0,0 +1,90 @@
+{% extends 'base/base.html' %}
+{% load static %}
+
+{% block title %}Resources{% endblock %}
+{% block head %}
+  <link rel="stylesheet" href="{% static "css/resources/resources.css" %}">
+{% endblock %}
+
+{% block content %}
+  {% include "base/navbar.html" %}
+
+  <section class="section">
+      <div class="container">
+          <div class="content">
+              <h1>Resources</h1>
+
+              <div class="tile is-ancestor">
+                  <a class="tile is-parent" href="/articles/category/guides">
```

Comment:
I'm aware this endpoint hasn't been implemented yet, but just making a note here so you're aware:

It's preferable to use django's [`{% url %}`](https://docs.djangoproject.com/en/3.1/ref/templates/builtins/#url) tag when referencing views, so we can reference it by namespace rather than the relative URL. When the `articles` endpoint exists we'll need to go back and use the url tag here.

Reply:
I this we should get this merged faster than content app one, as then I can make change there about it.